lib/administrate/page/collection.rb

```diff
@@ -21,6 +21,72 @@ def ordered_html_class(attr)
         ordered_by?(attr) && order.direction
       end
 
+      def scope_groups
+        if dashboard.collection_scopes.is_a?(Hash)
+          dashboard.collection_scopes.keys
+        else
+          dashboard.collection_scopes.any? ? [:scopes] : []
+        end
+      end
+
+      def scope_names(group = nil)
+        if dashboard.collection_scopes.is_a?(Hash)
+          group ||= dashboard.collection_scopes.keys.first
+          dashboard.collection_scopes[group].map &:to_s
+        else
+          dashboard.collection_scopes.map &:to_s
+        end.reject do |scope|
+          # do NOT show the wildcarded scopes
+          scope[-2..-1] == ':*'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.